qutip/entropy.py

```diff
@@ -113,11 +116,11 @@ def concurrence(rho):
     evals = rho_tilde.eigenenergies()
 
     # abs to avoid problems with sqrt for very small negative numbers
-    evals = abs(sort(real(evals)))
+    evals = abs(np.sort(np.real(evals)))
```

Comment:
Should the `abs` use numpy's?

Reply:
It wasn't using earlier and doesn't cause any issue with `jax`